tests/manage/monitoring/prometheusmetrics/test_monitoring_negative.py

```diff
@@ -66,6 +65,11 @@ def test_monitoring_shows_mon_down(measure_stop_ceph_mon):
         exp_good_time=measure_stop_ceph_mon['min_downtime'],
         exp_delay=expected_delay)
     mon_msg = "ceph_mon_quorum_status value should be affected by missing mon"
+
+    # checking validation results when both queries are performed makes sure
+    # that there is evidence for both mon and health queries in the test case
+    # logs in case of an assert failure
+    assert health_validation, health_msg
```

Comment:
I think you meant to check the health validation at this point?
In case you do, you'll need to call 'health_validation = check_query_range_result()' at line 44 here

Reply:
Function `check_query_range_result()` returns a boolean, and with this assert I'm making sure that it's true as expected. Moving this assert statement to the end will help us to see the logs from all queries in case any of these check boolean values are false.

Off course, it's possible that the result would be so wrong that some basic validation fails during either the query or the check, but in such a case it means we run into a much more serious problem.

Reply:
Also note that the useful evidence in the logs I have in mind is produced by logging in `check_query_range_result()` function.

Reply:
I see - thanks